docs/vectorizer-quick-start-voyage.md

```diff
@@ -0,0 +1,130 @@
+# Vectorizer quick start
+
+This page shows you how to create a vectorizer in a self-hosted Postgres instance. We also show how simple it is to do semantic search on the automatically embedded data!  
+We recommend to use voyage ai as an embedding provider, to follow this tutorial you need to have a voyage ai account and an api key. You can get one [here](https://voyageai.com/).
+
+## Setup a local development environment
+
+We use a docker-compose file to set up a development environment, it includes a:
+- Postgres deployment image with the TimescaleDB and pgai extensions installed
+- pgai vectorizer worker image
+
+On your local machine:
+
+1. **Create the Docker configuration for a local developer environment**
+
+   Create the following `docker-compose.yml` in a new directory:
+    ```yaml
+   name: pgai
+   services:
+     db:
+       image: timescale/timescaledb-ha:pg16
+       environment:
+         POSTGRES_PASSWORD: postgres
+         VOYAGE_API_KEY: your-api-key
+       ports:
+         - "5432:5432"
+       volumes:
+         - ./data:/var/lib/postgresql/data
+     vectorizer-worker:
+       image: timescale/pgai-vectorizer-worker:0.1.0
```

Comment:
Note the images need to be updated, not quite sure which release this will be. 

If you want to try this out use a docker-compose that builds the images like this right now:
```yaml
name: pgai
services:
  db:
    build:
      dockerfile: Dockerfile
      context: ../pgai/projects/extension
      target: pgai-test-db
    environment:
      POSTGRES_PASSWORD: postgres
      VOYAGE_API_KEY: your-api-key
    ports:
      - "5432:5432"
    volumes:
      - ./data:/var/lib/postgresql/data
  vectorizer-worker:
    build:
      dockerfile: Dockerfile
      context: ../pgai/projects/pgai
    environment:
      PGAI_VECTORIZER_WORKER_DB_URL: postgres://postgres:postgres@db:5432/postgres
      VOYAGE_API_KEY: your-api-key
    command: [ "--poll-interval", "5s" ]
```